src/pages/comparison/comparison.tsx

```diff
@@ -311,6 +311,36 @@ export const Comparison = ({ mode }: ComparisonPageProps) => {
     setLayersRightSide(bookmark.layersRightSide);
   };
 
+  const onDeleteBookmarkHandler = useCallback((bookmarkId: string) => {
+    setBookmarks((prev) =>
+      prev.filter((bookmark) => bookmark.id !== bookmarkId)
+    );
+  }, []);
+
+  const onEditBookmarkHandler = (bookmarkId: string) => {
+    createComparisonBookmarkThumbnail(
+      "#left-deck-container-wrapper",
+      "#right-deck-container-wrapper"
+    ).then((imageUrl) => {
+      if (!imageUrl) {
+        return;
+      }
```

Comment:
Create a Promise for this piece of code to reuse in adding and editing of bookmark